src/ContainerHeader.jsx

```diff
@@ -27,7 +27,7 @@
                     <TextInput id="containers-filter"
                                    placeholder={_("Type to filter…")}
                                    value={textFilter}
-                                   onChange={handleFilterChanged} />
+                                   onChange={(_, value) => handleFilterChanged(value)} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1299-20230606-080503-570e483c-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)